src/api.cpp

```diff
@@ -20,6 +21,7 @@
 // along with Rcpp.  If not, see <http://www.gnu.org/licenses/>.
 
 #define COMPILING_RCPP
+#define RCPP_USE_GLOBAL_ROSTREAM
```

Comment:
Should we `#ifndef` (or more properly `#if !defined()`) wrap this?  Or just not _in this segment_ ?  If so maybe add a comment.

Reply:
In order to offer the posibility of using a global `Rcout` to consumers, Rcpp itself must be compiled with this option. So given that this file is responsible for setting `COMPILING_RCPP`, I made it also responsible for setting `RCPP_USE_GLOBAL_ROSTREAM` (because it's also responsible for initializing the references to `Rcout` and `Rcerr` down below).